webpack/assets.js

```diff
@@ -0,0 +1,42 @@
+import WebpackIsomorphicToolsPlugin from 'webpack-isomorphic-tools/plugin';
+
+export default {
+  assets: {
+    images: {
+      extensions: ['jpeg', 'jpg', 'png', 'gif'],
+      parser: WebpackIsomorphicToolsPlugin.url_loader_parser
+    },
+    fonts: {
+      extensions: ['woff', 'woff2', 'ttf', 'eot'],
+      parser: WebpackIsomorphicToolsPlugin.url_loader_parser
+    },
+    svg: {
+      extension: 'svg',
+      parser: WebpackIsomorphicToolsPlugin.url_loader_parser
+    },
+    styles: {
+      extensions: ['css', 'less', 'sass', 'scss', 'styl'],
+      filter: (module, regex, options, log) => {
+        let filter = regex.test(module.name);
```

Comment:
I suggest writing is as
```js
const filter = options.development
  ? regex.test(module.name)
  : WebpackIsomorphicToolsPlugin.style_loader_filter(module, regex, options, log);
```

This project prefers `const` everywhere. Also I personally find it more readable than rebinding a variable.